lib/ueberauth.ex

```diff
@@ -69,7 +69,7 @@ defmodule Ueberauth do
   In phoenix setup a pipeline:
 
       pipeline :ueberauth do
-        Ueberauth.plug "/auth"
+        plug Ueberauth, base_path: "/auth"
```

Comment:
Love the `forward` approach.
On Fri, Nov 20, 2015 at 19:23 Chris McCord <notifications@github.com> wrote:

> In lib/ueberauth.ex
> <https://github.com/ueberauth/ueberauth/pull/7#discussion_r45538265>:
>
> > @@ -69,7 +69,7 @@ defmodule Ueberauth do
> >    In phoenix setup a pipeline:
> >
> >        pipeline :ueberauth do
> > -        Ueberauth.plug "/auth"
> > +        plug Ueberauth, base_path: "/auth"
>
> In that case, you could ditch the base_path option and rely on the
> conn.path_info/conn.request_path directly:
>
> scope "/" do
>   pipe_through [:browser, :ueberauth]
>   forward "/auth", Ueberauth
>
> end
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ueberauth/ueberauth/pull/7/files#r45538265>.
>